core/typedefs.h

```diff
@@ -316,4 +316,11 @@ struct BuildIndexSequence<0, Is...> : IndexSequence<Is...> {};
 #define ___gd_is_defined(val) ____gd_is_defined(__GDARG_PLACEHOLDER_##val)
 #define GD_IS_DEFINED(x) ___gd_is_defined(x)
 
+// Trivially constructible types are also zero-constructible.
+template <typename T>
+struct is_zero_constructible : std::is_trivially_constructible<T> {};
```

Comment:
In keeping parity with `std::is_trivially_constructible`, and type traits as a whole:
```suggestion
template <typename T>
struct is_zero_constructible : std::is_trivially_constructible<T> {};

template <typename T>
struct is_zero_constructible<const T> : is_zero_constructible<T> {};

template <typename T>
struct is_zero_constructible<volatile T> : is_zero_constructible<T> {};

template <typename T>
struct is_zero_constructible<const volatile T> : is_zero_constructible<T> {};
```

Reply:
Good call!
I assume you're getting the `std::is_trivially_constructible` trick from your system's headers? Mine are a lot more obscure and don't do anything like that. But it makes sense to me!